addons/hr_attendance/models/res_users.py

```diff
@@ -1,40 +1,22 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models, fields, _
+from odoo import models, fields
 
 
 class ResUsers(models.Model):
     _inherit = 'res.users'
 
-    hours_last_month = fields.Float(related='employee_id.hours_last_month')
-    hours_last_month_display = fields.Char(related='employee_id.hours_last_month_display')
     attendance_state = fields.Selection(related='employee_id.attendance_state')
     last_check_in = fields.Datetime(related='employee_id.last_attendance_id.check_in')
     last_check_out = fields.Datetime(related='employee_id.last_attendance_id.check_out')
```

Comment:
Is this the same as the other branch, a few fields not used anywhere? Separate commit I would say if we want to remove those. before the changes for instance